quantum/process_keycode/process_magic.c

```diff
@@ -47,7 +47,9 @@ bool process_magic(uint16_t keycode, keyrecord_t *record) {
             case MAGIC_TOGGLE_CONTROL_CAPSLOCK:
             case MAGIC_SWAP_ESCAPE_CAPSLOCK ... MAGIC_TOGGLE_ESCAPE_CAPSLOCK:
                 /* keymap config */
+#ifndef KEYMAP_CONFIG_RAM
```

Comment:
This might be better in the eeconfig_(read/update)_keymap function itself, since there are other places that these functions are called (bootmagic used to be one place, but the steno keycode in the planck, for instance).